tests/link/test_numba.py

```diff
@@ -317,18 +321,18 @@ def test_numba_box_unbox(input, wrapper_fn, check_fn):
 @pytest.mark.parametrize(
     "inputs, input_vals, output_fn, exc",
     [
-        (
-            [at.vector()],
-            [rng.standard_normal(100).astype(config.floatX)],
-            lambda x: at.sigmoid(x),
-            None,
-        ),
-        (
-            [at.vector() for i in range(4)],
-            [rng.standard_normal(100).astype(config.floatX) for i in range(4)],
-            lambda x, y, x1, y1: (x + y) * (x1 + y1) * y,
-            None,
-        ),
+        # (
+        #     [at.vector()],
+        #     [rng.standard_normal(100).astype(config.floatX)],
+        #     lambda x: at.sigmoid(x),
+        #     None,
+        # ),
+        # (
+        #     [at.vector() for i in range(4)],
+        #     [rng.standard_normal(100).astype(config.floatX) for i in range(4)],
+        #     lambda x, y, x1, y1: (x + y) * (x1 + y1) * y,
+        #     None,
+        # ),
```

Comment:
Did we mean to disable these cases?  If so, we should change them to `pytest.mark.xfail` cases and give a reason.